2023-11-report/index.md

```diff
@@ -0,0 +1,196 @@
+# Zürich 23.11 ZHF Hackathon and UX Workshop report
+
+On the last weekend of November 2023, the Zürich Friends of Nix hosted a multi-event around the [Zero Hydra Failures release preparations for NixOS 23.11](https://discourse.nixos.org/t/lets-have-a-great-23-11-release-cycle/33765).
+It featured a two-day hackathon and a UX workshop, that all took place at [Eastern Switzerland University of Applied Sciences](https://www.ost.ch/) (OST) right next to Lake Zürich.
+
+# Background and preparations
+
+The Nix ecosystem is growing, and so is the community building it.
+This can also be felt in the now regular Zürich ZHF hackathon, which has seen more and more participants over time.
+The event is designed to encourage people to learn to know each other, hack on everything Nix, and together shape the future of open source.
+
+The third iteration again incorporated participant feedback from last time, and also experience from in-person [workshops at NixCon 2023](https://discourse.nixos.org/t/nixcon-governance-workshop/32705):
+- More time for hacking and mingling
+- Shorter workshop sessions
+- A variety of brief talks for overview and inspiration
+- Increased coffee throughput
+
+Preparations started in June 2023, right after the event, by writing up a report, processing feedback, and sketching a schedule for the next event.
+In the four months before the event, we invited many guests and followed up with reminders, refined the agenda, and prepared some of the logistics.
+While originally there were ideas to make some progress on project governance topics, some key people couldn't make it this time and it turned out that the schedule wouldn't have allowed for it anyway.
+
+A brief survey on state the 6-months objectives from the last workshop showed that it wouldn't make sense to have a separate agenda item for that.
```

Comment:
```suggestion
A brief survey of 6-month-objectives from the last workshop showed that it wouldn't make sense to add this as a separate agenda item.
```

No, it quite literally didn't make sense because there was so little progress. We can't tell if there was or wasn't desire other than our own.